client/src/components/file/file.components.tsx

```diff
@@ -0,0 +1,15 @@
+import './file.styles.scss';
+
+import logo from '../../assets/logo.png'
+
+const FileComponent = () => {
+    return (
+        <div className="file-box">
+            <img src={logo} className="file-image" />
+            <span><strong>filename.pdf</strong></span>
```

Comment:
It is not really good practice to use `strong`.  It is better to use some CSS styling instead.